src/config.rs

```diff
@@ -302,4 +302,5 @@ create_config! {
     take_source_hints: bool, true, "Retain some formatting characteristics from the source code";
     hard_tabs: bool, false, "Use tab characters for indentation, spaces for alignment";
     wrap_comments: bool, false, "Break comments to fit on the line";
+    item_brace_style: BraceStyle, BraceStyle::SameLineWhere, "Brace style for structs and enums";
```

Comment:
I'm not sure the ordering makes much difference, but it makes sense to put this next to the function brace style option.